docs/index.md

```diff
@@ -19,18 +19,31 @@ GitHub integration is made through the GitHub app interface and the checks API (
 which allows results to be presented directly as inline annotations instead of
 a pass/fail status report.
 
-Precaution currently supports analysis of python files via Bandit and go files via Gosec. New languages may be added in future.
+Precaution currently supports analysis of:
+* Go files via [Gosec](https://github.com/securego/gosec)
+* JavaScript and TypeScript via [TSLint](https://github.com/palantir/tslint) and [tslint-config-security](https://github.com/webschik/tslint-config-security)
+* Python files via [Bandit](https://github.com/PyCQA/bandit)
+
+New languages may be added in future, please file an [issue](https://github.com/vmware/precaution/issues) for your language/linter of choice.
 
 * Documentation: [vmware/precaution/docs](https://vmware.github.io/precaution/)
 * Source: [vmware/precaution](https://github.com/vmware/precaution)
-* Bugs: [vmware/precaution/issues](https://github.com/vmware/precaution/issues)
+<!-- * Bugs: [vmware/precaution/issues](https://github.com/vmware/precaution/issues) -->
+
+## Try Precaution
 
-## Additional documentation
+Check out the several sections and you can continue with your project but with it being secure.
 
 - [Initial setup](initial_setup.md)
 - [Setting up Precaution as branch protection rule](branch_protection_rule.md)
 - [False positives and how to handle them](false_positivies.md)
+
+## Contributers documentation
+
+If you are enthusiastic about Precaution and want to improve it don't be concerned. Anyone can contribute, whether you’re new to the project or not.
+
+- Bugs: [vmware/precaution/issues](https://github.com/vmware/precaution/issues)
```

Comment:
Maybe we can move the `Bugs` link where are the Documentation and Source links because bugs can be written as well as our consumers and our contributors.
We could rephrase it to `Issues/Feature requests/Suggestions` which will invite our users to not only report bugs but also involve in the development of Precaution with ideas.

Reply:
Agreed. Filing issues/feature requests is a contribution, but we would like to encourage everyone to file them not just those who feel like they satisfy the label of contributor.